pkg/check/uploadstress/uploadstress.go

```diff
@@ -0,0 +1,108 @@
+package uploadstress
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"math/rand"
+	"sort"
+	"time"
+
+	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/check"
+	"github.com/ethersphere/beekeeper/pkg/random"
+	"golang.org/x/sync/errgroup"
+)
+
+// compile check whether UploadStress implements interface
+var _ check.Check = (*UploadStress)(nil)
+
+// UploadStress check
+type UploadStress struct{}
+
+// NewUploadStress returns new ping check
+func NewUploadStress() *UploadStress {
+	return &UploadStress{}
+}
+
+// Run executes upload stress check
+func (u *UploadStress) Run(ctx context.Context, cluster *bee.Cluster, o check.Options) (err error) {
+	buffer := 10
```

Comment:
I would name this variable `concurrency`, as it is used in the semaphore.

Reply:
Done.